shared/common-adapters/icon.constants.js

```diff
@@ -0,0 +1,265 @@
+// @flow
+// This file is GENERATED by npm run updated-fonts. DON'T hand edit
+
+export const FontIcons = {
```

Comment:
We use it to generate the dumb font sheet

On Wednesday, July 6, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/common-adapters/icon.constants.js
> <https://github.com/keybase/client/pull/3351#discussion_r69833175>:
>
> > @@ -0,0 +1,265 @@
> > +// @flow
> > +// This file is GENERATED by npm run updated-fonts. DON'T hand edit
> > +
> > +export const FontIcons = {
>
> Do we use it in code currently?
>
> I guess we can always change back to making an enum, just a bit wary of
> using $Enum and typeof on an object.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3351/files/82a835d604b001bbf6884f395c349cad2f400871#r69833175>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAfhU3_lbDP_nPniYH2qj0OvS2HesT21ks5qTEJ1gaJpZM4JGL0L>
> .
>